core/src/zeit/content/article/article.py

```diff
@@ -560,3 +572,21 @@ def get_dependencies(self):
         if audio_refs:
             return audio_refs.items
         return ()
+
+
+@grok.subscribe(
+    zeit.content.article.interfaces.IArticle, zeit.cms.checkout.interfaces.IBeforeCheckinEvent
+)
+def update_recipes_of_article(context, event):
+    if not FEATURE_TOGGLES.find('wcm_19_store_recipes_in_storage'):
+        return
+    if context.genre not in zeit.cms.config.get('zeit.wochenmarkt', 'recipe-genres', '').split(','):
+        return
+    recipes = context.body.filter_values(zeit.content.modules.interfaces.IRecipeList)
+    titles = []
+    ingredients = []
+    for recipe in recipes:
+        titles.append(recipe.title)
+        ingredients.extend(x.label for x in recipe.ingredients)
```

Comment:
```suggestion
        ingredients.extend(x.id for x in recipe.ingredients)
```
Die Labels sind doch nur für Anzeige. Artikel abfragen werden wir aber nach IDs, nachdem wir sie durch Substring/Prefixmatch/was auch immer ermittelt haben.